qa.gemspec

```diff
@@ -22,10 +22,13 @@ Gem::Specification.new do |s|
   s.add_dependency "nokogiri", "~> 1.6"
   s.add_dependency "activerecord-import"
   s.add_dependency "deprecation"
+  s.add_dependency "rdf", "2.2.3"
```

Comment:
I'm not sure I understand that.  It's 2.2.5 here: https://github.com/ruby-rdf/rdf/blob/develop/VERSION